addons/web/static/src/webclient/settings_form_view/settings_form_view.xml

```diff
@@ -6,11 +6,11 @@
         </xpath>
         <xpath expr="//Layout/t[@t-set-slot='layout-actions']" position="replace">
             <t t-set-slot="layout-actions">
-                <div class="o_cp_searchview d-flex flex-grow-1" role="search">
-                    <div class="o_searchview pb-1 align-self-center border-bottom flex-grow-1" role="search" aria-autocomplete="list">
-                        <i class="o_searchview_icon oi oi-search" role="img" aria-label="Search..." title="Search..." />
-                        <div class="o_searchview_input_container">
-                            <input type="text" class="o_searchview_input" accesskey="Q" placeholder="Search..." role="searchbox" t-model="searchState.value" t-ref="autofocus"/>
+                <div class="o_cp_searchview d-flex input-group" role="search">
+                    <div class="o_searchview form-control d-print-contents d-flex align-items-center py-1" role="search" aria-autocomplete="list">
+                        <i class="o_searchview_icon d-print-none oi oi-search me-2" role="img" aria-label="Search..." title="Search..." />
+                        <div class="o_searchview_input_container d-flex flex-grow-1 flex-wrap gap-1">
+                            <input type="text" class="o_searchview_input d-print-none flex-grow-1 w-auto border-0" accesskey="Q" placeholder="Search..." role="searchbox" t-model="searchState.value" t-ref="autofocus"/>
```

Comment:
What's the expected behavior for print?
In regular controlPanel we currently show the active tags only (no input's frame).
If we want to keep the same behaviour  here, just remove `d-print-none` from the  `<input>` and we're good 👍

Reply:
indeed it makes sense to show the search, since it is applied for search media as well